nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -249,6 +249,11 @@ language governing permissions and limitations under the License. -->
             <artifactId>super-csv</artifactId>
             <version>2.4.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.everit.json</groupId>
+            <artifactId>org.everit.json.schema</artifactId>
+            <version>1.4.0</version>
+        </dependency>
```

Comment:
In the pom file, you also have to add the json files you added for testing purpose in order to exclude those files during the rat check (licensing aspects). You can check that everything is fine by running a mvn clean install with the profile contrib-check:
````
mvn clean install -Pcontrib-check
````
Side note: from where are coming the json files your are using for tests? (just to be sure there is no issue)